src/networkremote/incomingmsg.h

```diff
@@ -0,0 +1,38 @@
+#ifndef INCOMINGMSG_H
+#define INCOMINGMSG_H
+
+#include <QObject>
+#include <QByteArray>
+#include <string>
```

Comment:
Should not use std::string, but QString instead.

Reply:
Fixed